src/components/EmptyDiary/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { ReactElement } from 'react';
+import styled from '@emotion/styled';
+import CardEmptyBackground from '@src/assets/CardEmpty.svg';
+import Logo from '@src/assets/Logo.png';
+import Plus from '@src/assets/Plus.svg';
+
+const Wrapper = styled.div`
+  position: relative;
+  width: 280px;
+  height: 512px;
+  filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
+
+  svg {
+    color: ${(props) => props.color};
+  }
+`;
+
+const ContentWrapper = styled.div`
+  position: absolute;
+  left: 135px;
+  top: 341px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  .plus-container {
+    margin-bottom: 37px;
+    position: relative;
+    img:last-child {
+      position: absolute;
+      left: 34px;
+      top: 17px;
+    }
+    span {
+      font-family: 'Gmarket Sans';
+      font-size: 16px;
+      color: ${({ theme }) => theme.colors.white100};
+    }
+  }
+`;
+
+const EmptyCard = (
+  <Wrapper>
+    <CardEmptyBackground />
+    <ContentWrapper>
+      <div className="plus-container">
+        <img src={Logo} alt="logo" />
+        <img src={Plus} alt="plus" />
```

Comment:
Plus는 svg라서 리액트 컴포넌트 호출하듯이 가져오면 될거 같아
```suggestion
        <Plus />
```

Reply:
```
img:last-child {
      position: absolute;
      left: 34px;
      top: 17px;
    }
```

제가 Plus absolute 써야해서 img 태그 사용했는데 컴포넌트로도 할 수 있나요?ㅠ

Reply:
div로 컴포넌트 감싸서 하면 되는건가요!

Reply:
그러면 위에서
`const StyledPlus = styled(Plus)```
해서 absolute 추가된 컴포넌트를 만들면 될거같아